library/src/animationManager.cxx

```diff
@@ -216,7 +216,7 @@ void animationManager::Tick()
     this->Importer->UpdateTimeStep(*this->CurrentTimeStep);
     this->Window->render();
 
-    this->CurrentTimeStep++;
+    ++this->CurrentTimeStep;
```

Comment:
what does this change ?

Reply:
cppcheck recommends using preincrement for non-trivial types. I recommend this video on the subject btw: https://www.youtube.com/watch?v=ObVRSNvGitE